Makefile

```diff
@@ -0,0 +1,22 @@
+IMAGE_NAME=morandirb
+CONTAINER_NAME=morandirb
+
+.PHONY: all run shell stop clean
+
+all: clean build run
```

Comment:
when mark set up the project it included a docker-compose file and I used it for a bit while I was working on this. I ended up removing it though, for a couple of reasons. firstly docker-compose by default doesn't give you an interactive prompt, which meant running in dettached mode, reattaching and running guard. this was too much faff for me, which is why I started to use make. once I had make, docker-compose didn't see to be adding anything much. which brings me to the second reason: to my mind docker-compose is really for composing services together. but morandi isn't even a service. it's a gem. that's why there's nothing to run in the dockerfile, so I run the tests instead.

with that said it would be easy to add docker-compose back in and even to provide both docker-compose and make, if you think it would be worth doing